python/tank/platform/qt/login.ui

```diff
@@ -0,0 +1,228 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>LoginDialog</class>
+ <widget class="QDialog" name="LoginDialog">
+  <property name="windowModality">
+   <enum>Qt::ApplicationModal</enum>
```

Comment:
This doesn't seem right. Not sure what implications it has got in terms of instantiation, but I don't think for example all of Maya should be blocked just because a user doesn't log in. This sounds like it could be a consequence of this, but I am not totally sure...